### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by azekillDIABLO** [LordStephen77/DreamBlocks-Beyond-of-Time#4]
Sorry for being so insistant :D 
AS you seeem to have much work with the game, i'm currently establishing the website! tell me if you want special things to appear!

**Comment by LordStephen77** [comment]
@azekillDIABLO I will publish the game here: https://itch.io/ is a very popular portal for indie games. Do not worry, the game will be developed later. As soon as I'm free I want to organize the easy ones for those who do not know the programming, to give everyone the ability to work without problems. Thank you so much for your interest.

**Comment by azekillDIABLO** [comment]
cool, it's a great thing to publish on itch.io! i have an account too! i will put a link to the itch.io page!

website first look: [https://github.com/azekillDIABLO/Dream-Blocks](https://github.com/azekillDIABLO/Dream-Blocks)

**Comment by LordStephen77** [comment]
@azekillDIABLO Cool! The site is perfect! I just wanted it so simple with a retro style game! You did a good job. When I put the credits into the game I'll put you in the foreground. Thank you so much I really like it.

**Comment by azekillDIABLO** [comment]
Thank you :) i still working on it! tell me when you need some changes! i'm honoured!

**Comment by LordStephen77** [comment]
@azekillDIABLO Very simple, enter the presentation text as you like, some screenshots of the game, then the rest I set it up. It's a nice job, though simple is what I wanted thanks.

**Comment by azekillDIABLO** [comment]
:D

**Comment by Gerold55** [comment]
Let me know if you need some ideas to add to the website I'm good at designing and creating concepts. So let me know plz.

**Comment by LordStephen77** [comment]
@Gerold55 You and azekillDIABLO are working on the website ok? Use html5 code. Make a simple website. AzekillDIABLO has made a small template perfect for the game. I recommend html5 tags. Thank you.

**Comment by azekillDIABLO** [comment]
Gerold, i'll be happy to work with you, just do some pull request! if you want to add something special, let me know!

**Comment by azekillDIABLO** [comment]
Site has been slightly improved!

**Comment by azekillDIABLO** [comment]
Website is finished, i'll just have to update things (i'm sure gerold will help me improving the content :smiley:)!

**Comment by Gerold55** [comment]
![screenshot 26](https://user-images.githubusercontent.com/13843095/27263045-9a0b84f4-5430-11e7-9e95-b5df4c9c96c7.png)

**Comment by tobyplowy** [comment]
@azekillDIABLO @Gerold55  no offense but that looks like a website from the 80s (in the bad way)

**Comment by Gerold55** [comment]
I know it's missing something but can't put my finger on it

**Comment by Gerold55** [comment]
![screenshot 27](https://user-images.githubusercontent.com/13843095/27263883-1b989ae2-5441-11e7-93af-97852243eb6f.png)

**Comment by tobyplowy** [comment]
@Gerold55 the new dream blocks logo at the top looks nice

Not sure about the grass blocks

**Comment by Gerold55** [comment]
@tobyplowy it's to show off what the game looks like a bit they might need 2 be brought down some

**Comment by tobyplowy** [comment]
@Gerold55 i will make a photoshop to show my idea

**Comment by tobyplowy** [comment]
@Gerold55 how about this
![27263883-1b989ae2-5441-11e7-93af-97852243eb6f](https://user-images.githubusercontent.com/16853304/27264461-51e63e72-547f-11e7-88f6-fa3545e77d7d.png)

**Comment by tobyplowy** [comment]
@Gerold55 maybe make the headder blue
![27263883-1b989ae2-5441-11e7-93af-97852243eb6f](https://user-images.githubusercontent.com/16853304/27264496-35668e04-5480-11e7-960b-7f2b00de06f2.png)

**Comment by Gerold55** [comment]
Try changing the header to a few layers of dirt on top and stone @ the bottom

**Comment by tobyplowy** [comment]
@Gerold55 that's just a Photoshop for demonstration purposes I dident program that :)

**Comment by Gerold55** [comment]
Yeah I know but wouldn't adding dirt and stone to the top look good instead of the blue?

**Comment by tobyplowy** [comment]
@Gerold55 I don't think so

**Comment by LordStephen77** [comment]
@Gerold55 You style is very beauty!

@tobyplowy 

> no offense but that looks like a website from the 80s (in the bad way)

That's what I wanted! I love the 80's! I want a game with the current style, old style. With the website you did a good job, as simple as I wanted! Simplicity is a good thing. I do not want complex content. The website is lightweight, easy to develop and will contain the essentials for the project (screenshots and description of the game, author, collaborators and social links). Good job!

**Comment by LordStephen77** [comment]
@tobyplowy Lo sfondo di Garold mi piace, togli il sole, e mantieni il tuo template con i pulsanti come i tuoi (grigi style pixel simili a Minecraft?). Decidete voi, state facendo un buon lavoro. Grazie mille ragazzi!

**Comment by Gerold55** [comment]
![screenshot 28](https://user-images.githubusercontent.com/13843095/27265479-b9a3750e-5464-11e7-8827-4948debbe599.png)

Does that look ok?

**Comment by tobyplowy** [comment]
@Gerold55 in my opinion there's way too much color again it looks like a website from the 80 (and again not in the good nostalgic way but in the ahh my eyes are melting way)

What bothers me (this is just my advice you don't have to follow it but it's recommended that you do)

1 Try to stick with grey and desaturated colours it is better and less painfull to look at and it looks more appealing also you need to use a color palette (my example has white black blue) your example has blue gray brouwn and yellow these colors don't really fit !
2 also please make that blue text black!
3 The cobblestone at the top doesn't really fit also it's too much repeated (texture is too small)
4 The dreamblocks logos shadow is white I'm sure that's not how shadows work :/

I guess that's it I hope I wasn't too critical remember I'm just giving feedback :)

**Comment by LordStephen77** [comment]
You manage the website, you are doing a great team work! this is the right spirit. Thank you all.

**Comment by azekillDIABLO** [comment]
that looks great! we should reuse the header of the game! i lovee this new background, send me it!
